src/main/java/ir/khu/jaobshaar/service/job/JobController.java

```diff
@@ -33,15 +33,16 @@ public JobController(JobManager jobManager) {
     @GetMapping("/employee")
     public ResponseEntity<List<JobDomain>> getEmployeeJobs(JobCriteria jobCriteria, Pageable pageable) {
         List<JobDomain> jobDomains = jobManager.getEmployeeJobs(jobCriteria, pageable);
-        ResponseEntity<List<JobDomain>> responseEntity = ResponseEntity.ok(jobDomains);
         HttpHeaders headers = new HttpHeaders();
         headers.add("total-count", "" + jobManager.countAll());
         return new ResponseEntity<>(jobDomains, headers, HttpStatus.OK);
     }
 
     @GetMapping("/employer")
     public ResponseEntity<List<JobDomain>> getEmployerJobs(Pageable pageable) {
-        return ResponseEntity.ok(jobManager.getEmployerJobs(pageable));
+        HttpHeaders httpHeaders = new HttpHeaders();
+        httpHeaders.add("total-count", "" + jobManager.countEmployerJobs());
```

Comment:
Please add the responsibility of creating your response to the job manager, and just call Response.ok(job manager).
this `countEmployerJobs` method is in job manager so , change access modifier to private and call it in the job manager too!.
@8fatemeT8